README.md

```diff
@@ -6,8 +6,10 @@ a simple & light 100 line package to call OpenAI, Azure, Cohere, Anthropic API E
 litellm manages:
 - translating inputs to completion and embedding endpoints
```

Comment:
@zakhar-kogan I made some updates to readme.md 
https://github.com/BerriAI/litellm

These include your changes, if the new readme looks good then you don't need to have this update on the PR

Reply:
Looks great to me!
Do I get it right I should just copy the new README & commit it?